ecies/ecies.go

```diff
@@ -103,10 +102,12 @@ func Decrypt(g kyber.Group, fn func() hash.Hash, priv kyber.Scalar, o *drand.ECI
 
 	// even though optional for this mode of ECIES, it _should_ not hurt if we
 	// add it.
-	nonce := make([]byte, 12)
-	if _, err := io.ReadFull(rand.Reader, nonce); err != nil {
-		return nil, err
-	}
+
+	// This nonce was never used in the original code; shall we omit generating it?
```

Comment:
What are your thoughts on this?

Reply:
Oh wow good catch, that looks like a bad copy paste from the encrypt function ;)

Reply:
Let's remove it entirely